templates/observability-remote-write-proxy.yml

```diff
@@ -0,0 +1,335 @@
+---
```

Comment:
Additional work will be needed for stage and production environments to integrate this template into them